app/build.gradle

```diff
@@ -15,6 +15,7 @@ android {
         versionCode 2
         versionName "1.0.1"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        multiDexEnabled true
```

Comment:
Why is this needed?

Reply:
True, I added it in to try and remove an error I was having earlier, slipped to removed.